vtex/utils/transform.ts

```diff
@@ -525,10 +525,10 @@ const toAdditionalPropertiesLegacy = (sku: LegacySkuVTEX): PropertyValue[] => {
   return [...specificationProperties, ...attachmentProperties];
 };
 
-const toOffer = ({ commertialOffer: offer, sellerId }: SellerVTEX): Offer => ({
+const toOffer = ({ commertialOffer: offer, sellerName }: SellerVTEX): Offer => ({
   "@type": "Offer",
   price: offer.spotPrice ?? offer.Price,
-  seller: sellerId,
+  seller: sellerName,
```

Comment:
```suggestion
const toOffer = ({ commertialOffer: offer, sellerId, sellerName }: SellerVTEX): Offer => ({
  "@type": "Offer",
  price: offer.spotPrice ?? offer.Price,
  seller: sellerId,
  sellerName,
```
This may be an option but it doesn't follow schema.org